### PR TITLE
changed http to https

### PR DESCRIPTION
**Pull request by Omvpatil**
- [x] Mark this if you consider it ready to merge
- [x] I've added tests (optional)
- [ ] I wrote some lines in the [book](https://github.com/radareorg/radare2book) (optional)

**Description**

**Copilot**

### CODE REVIEW

File: dist/cydia/deb_hand.mak [hunk 1]
```diff
@@ -126,4 +126,4 @@ clobber::
 		${PACKAGE_DIR}/data ${PACKAGE_DIR}/build
 
 push:
-	scp *.deb radare.org:/srv/http/radareorg/cydia/debs
+	scp *.deb radare.org:/srv/https/radareorg/cydia/debs
```
Comment (trufae) [inline dist/cydia/deb_hand.mak:129]: that change is not correct. the internal path in the server is http, not https

Comment (Omvpatil) [reply dist/cydia/deb_hand.mak:129]: ok

File: libr/arch/p/6502/plugin.c [hunk 2]
```diff
@@ -1,14 +1,14 @@
 /* radare - LGPL - Copyright 2019-2022 - condret, riq, pancake */
 
-/* 6502 info taken from http://unusedino.de/ec64/technical/aay/c64/bchrt651.htm
+/* 6502 info taken from https://unusedino.de/ec64/technical/aay/c64/bchrt651.htm
```
Comment (trufae) [inline libr/arch/p/6502/plugin.c:3]: this link doesnt work with https

**Comment by trufae** [comment]
hello?

File: libr/main/radare2.c [hunk 3]
```diff
@@ -191,7 +191,7 @@ static int main_help(int line) {
 		" -b [bits]    set asm.bits\n"
 		" -B [baddr]   set base address for PIE binaries\n"
 		" -c 'cmd..'   execute radare command\n"
-		" -C           file is host:port (alias for -c+=http://%%s/cmd/)\n"
+		" -C           file is host:port (alias for -c+=https://%%s/cmd/)\n"
```
Comment (trufae) [inline libr/main/radare2.c:194]:
```suggestion
		" -C           file is host:port (alias for -c+=http://%%s/cmd/)\n"
```

File: libr/main/radare2.c [hunk 4]
```diff
@@ -1160,7 +1160,7 @@ R_API int r_main_radare2(int argc, const char **argv) {
 		if (strstr (uri, "://")) {
 			r_core_cmdf (r, "=+ %s", uri);
 		} else {
-			argv[opt.ind] = r_str_newf ("http://%s/cmd/", argv[opt.ind]);
+			argv[opt.ind] = r_str_newf ("https://%s/cmd/", argv[opt.ind]);
```
Comment (trufae) [inline libr/main/radare2.c:1163]:
```suggestion
			argv[opt.ind] = r_str_newf ("http://%s/cmd/", argv[opt.ind]);
```

File: libr/main/r2agent.c [hunk 5]
```diff
@@ -20,7 +20,7 @@ static int usage(int v) {
 	"  -d        run in daemon mode (background)\n"
 	"  -h        show this help message\n"
 	"  -s        run in sandbox mode\n"
-	"  -u        enable http Authorization access\n"
+	"  -u        enable https Authorization access\n"
```
Comment (trufae) [inline libr/main/r2agent.c:23]:
```suggestion
	"  -u        enable http authorization access\n"
```

File: doc/zsh/_r2 [hunk 6]
```diff
@@ -16,7 +16,7 @@ _r2() {
   '-b+[set asm.bits]:bits:(8 16 32 64)'
   '-B+[set base address for PIE binaries]:baddr'
   '-c+[execute radare command]:cmd'
-  '-C[file is host:port (alias for -c+=http://%s/cmd/)]'
+  '-C[file is host:port (alias for -c+=https://%s/cmd/)]'
```
Comment (trufae) [inline doc/zsh/_r2:19]:
```suggestion
  '-C[file is host:port (alias for -c+=http://%s/cmd/)]'
```

File: dist/debian/deb_hand.mak [hunk 7]
```diff
@@ -134,4 +134,4 @@ clobber::
 		${PACKAGE_DIR}/data ${PACKAGE_DIR}/build
 
 push:
-	scp *.deb radare.org:/srv/http/radareorg/cydia/debs
+	scp *.deb radare.org:/srv/https/radareorg/cydia/debs
```
Comment (trufae) [inline dist/debian/deb_hand.mak:137]:
```suggestion
	scp *.deb radare.org:/srv/http/radareorg/cydia/debs
```

File: dist/cydia/deb_hand.mak (hunk 1 above)
Comment (trufae) [inline dist/cydia/deb_hand.mak:129]:
```suggestion
	scp *.deb radare.org:/srv/http/radareorg/cydia/debs
```

**Review by trufae — CHANGES_REQUESTED** [review]

File: doc/pdb/pdb_usage.md [hunk 8]
```diff
@@ -49,7 +49,7 @@ f pdb.__imp__printf = 0x192c8 # 0 .idata
 f pdb.__imp__system = 0x192c0 # 0 .idata
 f pdb._test_func__YAHHH_Z = 0x113e0 # 2 .text
 ...
-Check out this post for more information about pf: http://radare.today/types/
+Check out this post for more information about pf: https://radare.today/types/
```
Comment (trufae) [inline doc/pdb/pdb_usage.md:52]:
```suggestion
Check out this post for more information about pf: https://radareorg.github.io/blog/posts/types/
```

File: libr/arch/include/elf/common.h [hunk 9]
```diff
@@ -638,7 +638,7 @@
 #define DTF_1_PARINIT	0x00000001
 /* From
 
-   http://docs.sun.com:80/ab2/coll.45.13/LLM/@Ab2PageView/21165?Ab2Lang=C&Ab2Enc=iso-8859-1
+   https://docs.sun.com:80/ab2/coll.45.13/LLM/@Ab2PageView/21165?Ab2Lang=C&Ab2Enc=iso-8859-1
```
Comment (trufae) [inline libr/arch/include/elf/common.h:641]:
```suggestion
```
broken link

File: dist/windows/README.md [hunk 10]
```diff
@@ -1,6 +1,6 @@
 # radare2-installer
 Windows installer scripts for radare2
 
-[![Build Status](http://ci.rada.re/buildStatus/icon?job=radare2-win-installer)](http://ci.rada.re/job/radare2-win-installer/)
```
Comment (trufae) [inline dist/windows/README.md:4]: all those links are broken. ill update the readmes in a separate pr. but at least the pr should be good to merge now

File: test/README.md [hunk 11]
```diff
@@ -1,6 +1,6 @@
 # Radare2 Regression Test Suite
 
-A set of regression tests for Radare2 (http://radare.org).
+A set of regression tests for Radare2 (https://radare.org).
```
Comment (trufae) [inline test/README.md:3]:
```suggestion
A set of regression tests for Radare2 (http://radare.org).
```

**Pull request merged**